_topic_map.yml

```diff
@@ -269,3 +269,11 @@ Distros: openshift-acs
 Topics:
 - Name: Getting support for Red Hat Advanced Cluster Security for Kubernetes
   File: getting-support
+Name: Troubleshooting the Collector
```

Comment:
```suggestion
---
Name: Troubleshooting the Collector
```

Reply:
**issue:** Needs a YAML separator. 
**thought:** Maybe we can move this before the Support section so that Support appears as the last item.

Reply:
Ack